# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/RangeAnalysis.cpp
# Commit: 77ff4529ea3e
# Full Hash: 77ff4529ea3eb01ca0013bd4747979c226fb617f
# Author: Steve Fink <sfink@mozilla.com>
# Date: 2021-08-13 09:27:46
# Regressor Bug: 1720422
# File Overlap Count: 2
# Description:
#   Bug 1720422 - Expose a templatized TypedArray<T> class in JSAPI r=jonco,nbp
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119853
# ==============================================================================

diff -r 16fa627201d1 -r 77ff4529ea3e js/src/jit/RangeAnalysis.cpp
--- a/js/src/jit/RangeAnalysis.cpp	Thu Aug 12 20:28:53 2021 +0000
+++ b/js/src/jit/RangeAnalysis.cpp	Thu Aug 12 20:44:14 2021 +0000
@@ -1815,7 +1815,7 @@
 void MTypedArrayElementSize::computeRange(TempAllocator& alloc) {
   constexpr auto MaxTypedArraySize = sizeof(double);
 
-#define ASSERT_MAX_SIZE(T, N)                   \
+#define ASSERT_MAX_SIZE(_, T, N)                \
   static_assert(sizeof(T) <= MaxTypedArraySize, \
                 "unexpected typed array type exceeding 64-bits storage");
   JS_FOR_EACH_TYPED_ARRAY(ASSERT_MAX_SIZE)